api/app/controllers/sessions_controller.rb

```diff
@@ -17,7 +17,7 @@ def register
     if user.save
       render json: user
     else
-      render json: { errors: user.errors}, status: :bad_request
+      render json: { errors: user.errors.full_messages}, status: :bad_request
```

Comment:
Could you please explain what are you trying to do with `.full_messages`?

Reply:
It returns an array of strings with all the errors